network-ntt/src/reply_handle.rs

```diff
@@ -0,0 +1,120 @@
+//! This module provides an abstraction to a simple connection
+//! reply mechanism. NT provides a very cheap lightweight connection,
```

Comment:
I intended for `network-core` future and stream implementations to hide the details of serving the request. The `Server` implementation should be able to resolve requests with its embedded `Node` object.